addons/mass_mailing/static/src/scss/mass_mailing.scss

```diff
@@ -39,10 +39,19 @@
         margin-top: $o-sheet-cancel-tpadding + 33px;
     }
     .o_FormRenderer_chatterContainer {
+        --ChatterTopbar-padding-left: 0;
+        --ChatterContainer-overflow: visible;
+        --ChatterTopbar-border: none;
+        --ChatterActivityBox_title-margin: #{map-get($spacers, 1)};
+
         margin: 0;
         max-width: unset;
         width: 100%;
         padding: 0;
+
+        .o_MessageList_separator:first-child {
+            margin-top: -#{map-get($spacers, 3)};
+        }
```

Comment:
Hello, thank you very much for your review!

I'll push the requested changes, however for the `.o_MessageList_separator:first-child` applying the negative margin on the parent `.o_MessageList` creates an overlap: 
![image](https://user-images.githubusercontent.com/118886338/229108978-cab457e2-75ff-4469-a34d-88ddce878a29.png)

The goal is to counter the existing padding on the `.o_MessageList_separator`. The best working solution I could find for now, is [this trick](https://stackoverflow.com/questions/2717480/css-selector-for-first-element-with-class/8539107#8539107) that allows to emulate a `first-of-class`. 

Which would look similar to this in our case:

`        .o_MessageList_separator {
            margin-top: -#{map-get($spacers, 3)};
        }

        .o_MessageList_separator ~ .o_MessageList_separator {
            margin-top: unset;
        }
`
But it doesn't really fix the issue since if we are changing the layout by putting the message before the separator, we would probably want a different spacing.

I think we should stick with the first-child for now and adapt it later if we change the layout. 
lmk what you think!